src/hooks/api/useCashFlow.ts

```diff
@@ -0,0 +1,47 @@
+import { DateTime, Interval } from 'luxon';
+import { useQuery } from '@tanstack/react-query';
+import type { UseQueryResult } from '@tanstack/react-query';
+
+import { Split } from '@/book/entities';
+import { useInterval } from '@/hooks/state';
+
+/**
+ * Retrieve the cashflow for a given account in the specified date.
+ *
+ * Returns an array containing the account guid, the total for that account, the type and
+ * the name.
+ */
+export function useCashFlow(
+  account: string,
+  selectedInterval?: Interval,
+): UseQueryResult<{ guid: string, total: number, type: string, name: string }[]> {
+  const { data: defaultInterval } = useInterval();
+  const interval = selectedInterval || defaultInterval;
```

Comment:
nothing changed except this where we can pass a custom interval or use `useInterval` by default